tests/micro/arduino/conftest.py

```diff
@@ -0,0 +1,97 @@
+import datetime
+import pathlib
+
+import pytest
+
+import tvm.target.target
+
+# The models that should pass this configuration. Maps a short, identifying platform string to
+# (model, zephyr_board).
+PLATFORMS = {
+    "due": ("sam3x8e", "due"),
+    "feathers2": ("esp32", "feathers2"),
+    "nano33ble": ("nrf52840", "nano33ble"),
+    "pybadge": ("atsamd51", "pybadge"),
+    "spresense": ("cxd5602gg", "spresense"),
+    "teensy40": ("imxrt1060", "teensy40"),
+    "teensy41": ("imxrt1060", "teensy41"),
+}
+
+TEMPLATE_PROJECT_DIR = (
+    pathlib.Path(__file__).parent
+    / ".."
+    / ".."
+    / ".."
+    / "apps"
+    / "microtvm"
+    / "arduino"
+    / "template_project"
+).resolve()
+
+
+def pytest_addoption(parser):
+    parser.addoption(
+        "--microtvm-platforms",
+        default=["due"],
+        nargs="*",
+        choices=PLATFORMS.keys(),
+        help="Target platforms for microTVM tests.",
+    )
+    parser.addoption(
+        "--arduino-cli-cmd",
+        default="arduino-cli",
+        help="Path to `arduino-cli` command for flashing device.",
+    )
+    parser.addoption(
```

Comment:
i'd almost propose naming this --test-build-only and add this flag to tests/scripts/task_python_microtvm.sh to make it obvious we aren't running the full test in CI

Reply:
I like that idea - changed.